src/rma/js/components/destruction-list/zaken-table.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import { CheckboxInput } from "./inputs";
+
+
+function displayZaaktype (zaaktype) {
+    return `${zaaktype.omschrijving} (${zaaktype.versiedatum}}`;
+}
+
+
+function ZakenTable(props) {
+    const { zaken, isLoaded, error, checkboxes, setCheckboxes } = props;
+
+    const [selectAll, setSelectAll] = useState(false);
```

Comment:
Works for now - but you can refactor the component trees so that all the state lives in the root node and all children are prop-driven. Which helps in preventing components going out of sync with each other if there are bugs.

Reply:
added to TODO